dask_jobqueue/core.py

```diff
@@ -202,9 +206,6 @@ def __init__(
 
         if interface:
             extra = extra + ["--interface", interface]
-            kwargs.setdefault("host", get_ip_interface(interface))
```

Comment:
After having looked a bit more in details, I am reasonably sure that `host` in kwargs is not used at all, i.e. this is another manifestation of #386. 

This code was introduced in https://github.com/dask/dask-jobqueue/pull/135. At that time the `kwargs` were passed to `LocalCluster`. Nowadays the kwargs are passed to the `job_cls`. This does not make any sense to pass `host=something` to the worker arguments where `something` is computed where on the node the scheduler runs (similar comment as https://github.com/dask/dask-jobqueue/issues/382#issuecomment-596915307) ...